modules/auth_email/module.json

```diff
@@ -0,0 +1,81 @@
+{
+	"name": "Authentication",
+	"description": "Authenticate users with multiple authentication methods.",
+	"icon": "key",
+	"tags": [
+		"core",
+		"auth",
+		"user"
+	],
+	"authors": [
+		"rivet-gg",
+		"NathanFlurry"
+	],
+	"status": "stable",
+	"dependencies": {
+		"email": {},
+        "identities": {},
+        "users": {},
+		"tokens": {},
+		"user_passwords": {},
+		"rate_limit": {}
+	},
+	"scripts": {
```

Comment:
this api is suuuper confusing (i know i helped design it :P). thoughts on splitting this up in to two modules which expose public scripts that call internal scripts exposed in this module. so devs will call `auth_email_password` and `auth_email_passwordless`.

Reply:
Will do eventually. Soon™.